### ISSUE TITLE
@authorize decorator drops the skip flag after the first call in a Controller.

### ISSUE THREAD

**Issue by yshing** [strongloop/loopback-next#7090]
## Steps to reproduce
Using the following test case. It seems the `skip` flag is only kept for the first method decorated using @authorize

@loopback/authorization version: 0.7.6

Running the following test fails.

```
import {
  AuthorizationMetadata,
  AUTHORIZATION_METHOD_KEY,
  authorize,
} from '@loopback/authorization';
import {MetadataInspector} from '@loopback/metadata';
import {get, post} from '@loopback/rest';
import {expect} from '@loopback/testlab';

class TestController {
  @authorize({skip: true})
  @get('/{id}')
  async updateOne() {}

  @authorize({
    allowedRoles: ['admin'],
    skip: true,
  })
  @get('/')
  async update() {}

  @authorize({skip: true})
  @post('/{id}/test')
  async anotherOne() {}

  @authorize({skip: true})
  @post('/test')
  async anotherRoot() {}

  @authorize.skip()
  @get('/some-unauthorized-endpoint')
  async usingAuthorizeDotSkip() {}
}

describe('Plain Controller Metadata', () => {
  let controller: TestController;
  before(async () => {
    controller = new TestController();
  });

  it('AuthorizationMetadata Test', async () => {
    const metadata = MetadataInspector.getAllMethodMetadata<AuthorizationMetadata>(
      AUTHORIZATION_METHOD_KEY,
      controller,
    );
    expect(metadata).to.eql({
      update: {
        skip: true,
      },
      updateOne: {
        skip: true,
      },
      anotherOne: {
        skip: true,
      },
      anotherRoot: {
        skip: true,
      },
      usingAuthorizeDotSkip: {
        skip: true,
      },
    });
  });
});

```

## Current Behavior
skip: true Flag is only kept for the first method in the Controller..
```
      + expected - actual

       {
      -  "anotherOne": {}
      -  "anotherRoot": {}
      +  "anotherOne": {
      +    "skip": true
      +  }
      +  "anotherRoot": {
      +    "skip": true
      +  }
         "update": {
      -    "allowedRoles": [
      -      "admin"
      -    ]
      +    "skip": true
         }
         "updateOne": {
           "skip": true
         }
      -  "usingAuthorizeDotSkip": {}
      +  "usingAuthorizeDotSkip": {
      +    "skip": true
      +  }
       }
```
## Expected Behavior

The test should pass..

## Link to reproduction sandbox

Above test case should be obvious enough?

## Additional information
This causes the every method after the first one declared with `@authorize({skip: true})` in the controller fail to execute properly.

**Comment by yshing** [comment]
Seems like fixed by #7073. but not released yet.

**Comment by yshing** [comment]
This is fixed after the changes in #7073 is released.
